readthedocs/analytics/signals.py

```diff
@@ -0,0 +1,32 @@
+"""Django signals for the analytics app."""
+from django.db.models import F
+from django.dispatch import receiver
+from django.utils import timezone
+
+from readthedocs.api.v2.signals import footer_response
+from readthedocs.projects.models import Feature
+
+from .models import PageView
+
+
+@receiver(footer_response)
+def increase_page_view_count(sender, **kwargs):
+    """Increase the page view count for the given project."""
+    del sender  # unused
+    context = kwargs['context']
+
+    project = context['project']
+    version = context['version']
+    # footer_response sends an empty path for the index
+    path = context['path'] or '/'
```

Comment:
I imagine we might want to catch `/index.html` and `/` as being the same at some point, but seems overcomplicated for now.

Reply:
I actually believe this code will handle it correctly.

Reply:
It doesn't. This change now stores the exact path requested.